spec/controllers/image_sets_controller_spec.rb

```diff
@@ -88,4 +88,18 @@
         change { image_set.reload.main_image }.from(nil).to(main_image)
     }
   end
+
+  describe '#destroy' do
+    let(:image_set) { Fabricate :image_set_with_images }
+    let(:request) { -> { delete :destroy, id: image_set.id } }
+
+    it_behaves_like "an authenticated controller"
+    it { expect { subject }.to change { ImageSet.count }.by(1) }
+    it { expect { subject }.to change { Image.count }.by(5) }
+
```

Comment:
@redyaffle not sure this is testing the right thing. we want ImageSet.count to change by **-1** and Image.count by **-5**. This test is actually passing because the objects are getting created and not deleted. I think if you get rid of the `require_organization` filter above then ImageSet.count will go down by -1. Image.count will not go down by -5 unless we add `dependent: destroy' for images in the image_set definition, which we should do anyway.